### PR TITLE
solution added

### PR DESCRIPTION
**Pull request by joynewton1996**

### CODE REVIEW

File: hacker_Rank/May17_Answer.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def solution(A):
+    non_repeat = []
+    for num in a:
+        if num not in non_repeat:
+            non_repeat.append(num)
```
Comment (roy-einstein) [inline hacker_Rank/May17_Answer.py]: can we use ``` set() ``` here,

**Review by roy-einstein — CHANGES_REQUESTED** [review]
Logic is wrong

File: hacker_Rank/May17_Answer.py (hunk 1 above)
Comment (joynewton1996) [reply hacker_Rank/May17_Answer.py]: okay

**Review by roy-einstein — APPROVED** [review]
good work

**Pull request merged**